sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/XmlExternalEntityShouldNotBeParsed_XmlDocument_Net35.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Configuration;
+using System.IO;
+using System.Text;
+using System.Xml;
+using System.Xml.Linq;
+using System.Xml.Resolvers;
+using System.Xml.XPath;
+using System.Xml.Xsl;
+using Microsoft.Web.XmlTransform;
+
+namespace Tests.Diagnostics
+{
+    class Test
```

Comment:
I think we should add also a compliant case for .Net 3.5

Reply:
and how could we test .NET 3.5?